tests/functional/spec/features/routing/date.spec.js

```diff
@@ -118,22 +125,35 @@ describe("Feature: Routing on a Date", () => {
   describe("Greater Than", () => {
     describe("Given I start date routing greater than survey", () => {
       beforeEach(() => {
-        browser.openQuestionnaire("test_routing_date_greater_than.json");
+        browser.openQuestionnaire("test_new_routing_date_greater_than.json");
       });
 
-      it("When I enter a date greater than March 2017, Then I should be routed to the correct page", () => {
-        $(DateGreaterThanQuestionPage.Month()).setValue(4);
-        $(DateGreaterThanQuestionPage.Year()).setValue(2017);
+      it("When I enter a date greater than the 1st March 2017, Then I should be routed to the correct page", () => {
+        $(DateGreaterThanQuestionPage.day()).setValue(2);
+        $(DateGreaterThanQuestionPage.month()).setValue(3);
+        $(DateGreaterThanQuestionPage.year()).setValue(2017);
         $(DateGreaterThanQuestionPage.submit()).click();
 
         const expectedUrl = browser.getUrl();
 
         expect(expectedUrl).to.contain(CorrectAnswerPage.pageName);
       });
 
-      it("When I enter a date less than or equal to March 2017, Then I should be routed to the incorrect page", () => {
-        $(DateGreaterThanQuestionPage.Month()).setValue(3);
-        $(DateGreaterThanQuestionPage.Year()).setValue(2017);
+      it("When I enter the 1st March 2017, Then I should be routed to the incorrect page", () => {
```

Comment:
Although this wasn't supposed to be changed I could leave an or condition in a test that wasn't exercising both parts, so I added an extra test. I also need to change the type as it was MonthYear